transforms/src/button-v0-to-v1.ts

```diff
@@ -0,0 +1,110 @@
+import { Transform, JSCodeshift, FileInfo, API, Options, ImportDeclaration, ASTPath, JSXElement } from 'jscodeshift';
+
+export const transform: Transform = (fileInfo: FileInfo, api: API, options: Options) => {
+  const j: JSCodeshift = api.jscodeshift;
+
+  const printOptions = options.printOptions || {
+    printWidth: 140,
+    quote: 'single',
+    tabWidth: 2,
+    trailingComma: true,
+  };
+
+  const root = j(fileInfo.source);
+  root
+    // Find all imports that import from '@fluentui/react-native' or '@fluentui-react-native/button'
+    .find(
+      j.ImportDeclaration,
+      (path: ImportDeclaration) => path.source.value === '@fluentui/react-native' || path.source.value === '@fluentui-react-native/button',
+    )
+    .forEach((path: ASTPath<ImportDeclaration>) => {
+      if (!path.value.specifiers) {
+        return;
+      }
+
+      // Find the first import specifier that imports Button, PrimaryButton or StealthButton.
+      const buttonSpecifier = path.value.specifiers.findIndex(
+        (specifier) =>
+          specifier.type === 'ImportSpecifier' &&
+          (specifier.imported.name === 'Button' ||
+            specifier.imported.name === 'PrimaryButton' ||
+            specifier.imported.name === 'StealthButton'),
+      );
+
+      // Change that import to be 'ButtonV1 as Button'
+      if (buttonSpecifier !== undefined && buttonSpecifier !== -1) {
+        path.value.specifiers[buttonSpecifier] = j.importSpecifier(j.identifier('ButtonV1'), j.identifier('Button'));
+      }
```

Comment:
I guess it'll be useful to have another transform int he future when we eventually rename "ButtonV1" to just "Button".

Reply:
Yeah, I have a local copy of that, I haven't converted it into TS yet.